egs2/TEMPLATE/svs1/scripts/utils/combine_data.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
```

Comment:
This is a modified version of kaldi combine data which has support to music data. We can discuss if it is necessary